modules/k6/k6.go

```diff
@@ -0,0 +1,95 @@
+package k6
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+// K6Container represents the K6 container type used in the module
+type K6Container struct {
+	testcontainers.Container
+}
+
+// WithTestScript mounts the given script into the ./test directory in the container
+// and passes it to k6 as the test to run.
+// The path to the script must be an absolute path
+func WithTestScript(scriptPath string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) {
+		script := filepath.Base(scriptPath)
+		target := fmt.Sprintf("/tests/%s", script)
+		mount := testcontainers.ContainerMount{
+			Source: testcontainers.GenericBindMountSource{
+				HostPath: scriptPath,
+			},
+			Target: testcontainers.ContainerMountTarget(target),
+		}
+		req.Mounts = append(req.Mounts, mount)
```

Comment:
Let's use req.Files instead of Mounts. Why? Because Files are copied to the container, and mounts not. That will make it work with remote Docker engines.

Reply:
Maybe I'm not understanding something about mount vs copy, but the idea is that the cache will be modified by the container and those changes will be available for the next container execution. This is important when running multiple tests, for example in a table-driven test.

Reply:
I think the script files must be copied. The cache path, this is the one I'd like to understand better (see https://github.com/testcontainers/testcontainers-go/pull/1721#discussion_r1350115875)

Reply:
My bad. I though the comment was about the cache. :pray: